build/Linux+BSD/portable/copy-libs

```diff
@@ -72,7 +74,7 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sensors.so.5
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
-  copyLib libQt5WebChannel.so.5
```

Comment:
@shoogle, I finally got the travis build to work on both for x86 and cross armhf.  Here is output of check depends of the [x86-64 build](https://travis-ci.org/ericfont/MuseScore/jobs/118950905) when running on my non-fresh arch linux x86-64 machine:

[20160328_armcross_travisx86-64_check-depends_non-fresh-archlinux.txt](https://github.com/musescore/MuseScore/files/191728/20160328_armcross_travisx86-64_check-depends_non-fresh-archlinux.txt)

Unfortunately, I do see libQt5WebChannel.so.5 in the provided by "System only" list, which suggests to me that somehow we do need to be able to include this library.  But the app image executes just fine, and I can't tell anything is wrong, except that I see the following message in console:<pre>QObject::connect: Cannot connect (null)::stateChanged(QNetworkSession::State) to QNetworkReplyHttpImpl::_q_networkSessionStateChanged(QNetworkSession::State)
</pre>
whether that can be safely ignored, I don't know.  What exactly are we using webkit for, considering we don't the musescore connect explorer?